src/Translation/Translator.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Translation;
+
+use Contao\CoreBundle\Framework\ContaoFrameworkInterface;
+use Contao\System;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Translator.
+ *
+ * @author Martin Auswöger <martin@auswoeger.com>
+ */
+class Translator implements TranslatorInterface
```

Comment:
Since this Translator is only for usage with  `$Globals['TL_LANG']` and hopefully replaced someday by a solution which doesn't need to use a superglobal I would recommend a name like `GlobalsTranslator` or so

Reply:
Where this translator gets the messages from may change in the future. In the optimal case this class gets removed in the future and the default translater service can be used.